tutorials/nm_example.py

```diff
@@ -0,0 +1,65 @@
+import pathlib
```

Comment:
Can you please double-check the tutorial? The propagator used was called ``AdaptedNM``. I assume this should be the ``ParallelNelderMead`` propagator. Also, the tutorial was based on an outdated version. I modified parameter names etc. to make it work on my machine. Can you check whether it also works for you as expected?